tutorials/darcy_thermo_mech/doc/content/systems/kernels.md

```diff
@@ -36,39 +36,10 @@ Current quadrature point index
 
 !---
 
-## Kernel Base Classes
+## Kernel Base Class
 
 | Base | Override | Use |
 | :- | :- | :- |
 | Kernel\\ +ADKernel+ | computeQpResidual | Use when the term in the [!ac](PDE) is multiplied by both the test function and the gradient of the test function (`_test` and `_grad_test` must be applied) |
-| KernelValue\\ +ADKernelValue+ | precomputeQpResidual | Use when the term computed in the [!ac](PDE) is only multiplied by the test function (do not use `_test` in the override, it is applied automatically) |
-| KernelGrad\\ +ADKernelGrad+ | precomputeQpResidual | Use when the term computed in the [!ac](PDE) is only multiplied by the gradient of the test function (do not use `_grad_test` in the override, it is applied automatically) |
```

Comment:
I don't want to get rid of these. This is good information to present so when they run across them they understand what they are. I also will use them for the LaplaceYoug.

Reply:
I reverted this change. As we discussed in the video call, I'll speak about `ADDiffusion` as a standalone example of usage and brief intro to those specialized classes. Then we can move into `DarcyPressure` as an example of standard `ADKernel`